src/verify.c

```diff
@@ -316,6 +316,26 @@ static struct list *download_loop(struct list *files, int isfailed)
 		}
 
 		if (hash_needs_work(file, local.hash)) {
+			/* Mix content is local, so don't queue files up for curl downloads */
+			if (file->is_mix) {
+				char *filename;
+				char *url;
+				string_or_die(&url, "%s/%i/files/%s.tar", MIX_STATE_DIR, file->last_change, file->hash);
```

Comment:
Not a URL. Needs a better variable name.

Reply:
Similarly for this comment, I'm going to ignore my concern about variable-name appropriateness for now.